db/migrate/20221019162507_create_db_sprint_tasks.rb

```diff
@@ -0,0 +1,9 @@
+class CreateDBSprintTasks < ActiveRecord::Migration[7.0]
+  def change
+    create_table :sprint_tasks do |t|
+      t.integer :task_id
+      t.integer :sprint_id
+      t.string :data
```

Comment:
PostgreSQL wspiera przechowywanie i szukanie po JSON.
Zrobiłem research i lepiej jakby pole `data` było typu `:jsonb`.

Będzie o wiele szybsze i będziemy mogli normalnie robić `where` po zawartości tego JSONa.

https://blog.saeloun.com/2022/01/04/postgresql-data-types-in-rails.html